flink-table/flink-table-planner/src/test/resources/org/apache/flink/table/planner/plan/rules/logical/ProjectWindowTableFunctionTransposeRuleTest.xml

```diff
@@ -16,7 +16,43 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 <Root>
-  <TestCase name="testNoColumnToPrune">
+  <TestCase name="testNoColumnToPrune[isStreamingMode=false]">
```

Comment:
just add tests for project-transpose in WindowTableFunctionTest which is enough I think. We need e2e plan test for this case instead of function test for ProjectWindowTableFunctionTransposeRule

Reply:
Existed UT `WindowTableFunctionTest#testCascadingWindowAggregate` in `WindowTableFunctionTest` already cover this case, you could find a projection node after TableSourceScan.
We could find this way is not so explicit.
I think there is no harm to add extra UTs in `ProjectWindowTableFunctionTransposeRuleTest`